test/integration/consul-container/test/gateways/gateway_endpoint_test.go

```diff
@@ -150,6 +150,47 @@ func createCluster(t *testing.T, ports ...int) *libcluster.Cluster {
 	return cluster
 }
 
+func createGateway(gatewayName string, protocol string, listenerPort int) *api.APIGatewayConfigEntry {
+	return &api.APIGatewayConfigEntry{
+		Kind: "api-gateway",
+		Name: gatewayName,
+		Listeners: []api.APIGatewayListener{
+			{
+				Name:     "listener",
+				Port:     listenerPort,
+				Protocol: protocol,
+			},
+		},
+	}
+}
+
+func checkGatewayConfigEntry(t *testing.T, client *api.Client, gatewayName string, namespace string) {
+	require.Eventually(t, func() bool {
+		entry, _, err := client.ConfigEntries().Get(api.APIGateway, gatewayName, &api.QueryOptions{Namespace: namespace})
+		assert.NoError(t, err)
+		if entry == nil {
+			return false
+		}
+		apiEntry := entry.(*api.APIGatewayConfigEntry)
+		t.Log(entry)
+		return isAccepted(apiEntry.Status.Conditions)
+	}, time.Second*10, time.Second*1)
```

Comment:
Would usually expect to use test helpers in these tests
```suggestion
	require.Eventually(t, func() bool {
		entry, _, err := client.ConfigEntries().Get(api.APIGateway, gatewayName, &api.QueryOptions{Namespace: namespace})
		require.NoError(t, err)
		require.NotNil(t, entry)

		apiEntry, ok := entry.(*api.APIGatewayConfigEntry)
		require.True(t, ok)
		
		t.Log(entry) // Mean to leave this here?
		return isAccepted(apiEntry.Status.Conditions)
	}, time.Second*10, time.Second*1)
```

Reply:
Agreed, looks like I just missed updating it here.